cmd/xcodeUITests.go

```diff
@@ -0,0 +1,141 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/goinp/goinp"
+	"github.com/bitrise-tools/codesigndoc/codesigndocuitests"
+	"github.com/bitrise-tools/codesigndoc/xcodeuitest"
+	"github.com/bitrise-tools/go-xcode/utility"
+	"github.com/spf13/cobra"
+)
+
+var xcodeUITestsCmd = &cobra.Command{
+	Use:   "xcodeuitests",
+	Short: "Xcode project scanner for UI tests",
+	Long:  `Scan an Xcode project for UI test targets`,
+
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE:          scanXcodeUITestsProject,
+}
+
+func init() {
+	scanCmd.AddCommand(xcodeUITestsCmd)
+
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodeProjectFilePath, "file", "", "Xcode Project/Workspace file path")
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodeScheme, "scheme", "", "Xcode Scheme")
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodebuildSDK, "xcodebuild-sdk", "", "xcodebuild -sdk param. If a value is specified for this flag it'll be passed to xcodebuild as the value of the -sdk flag. For more info about the values please see xcodebuild's -sdk flag docs. Example value: iphoneos")
```

Comment:
the definition of the variables paramXcodeProjectFilePath (xcode.go line 31) should be duplicated here?

Reply:
They are in the same package: `cmd`. Because of this there is no need to duplicate them.
And I could not "duplicate" them on the same package without changing the variables' name.